src/plugins/content_management/common/rpc/msearch.ts

```diff
@@ -45,6 +46,12 @@ export interface MSearchIn {
 
 export type MSearchResult<T = unknown> = SearchResult<T>;
 
+export interface MSearchIn<Options extends void | object = object> {
+  contentTypes: Array<{ contentTypeId: string; version?: Version }>;
+  query: MSearchQuery;
+  options?: Options;
```

Comment:
I agree that `options` make sense, but are they actually used anywhere for `msearch` in this pr? It seems they are not used and, as I understand, they won't work because the implementation of the `msearch` service wasn't changed to work with `options`. Or am I missing something?

Reply:
Looking at the implementation it seems it's dropped. I can drop it here at `search` wrapping function.

Reply:
got it. I suggest we revert these changes to msearch interface, because this can't used in msearch